lib/components/user/new-account-wizard.js

```diff
@@ -40,44 +40,41 @@ class NewAccountWizard extends Component {
       // and thus doesn't carry a nextId.
       verify: {
         hideNavigation: true,
-        pane: VerifyEmailPane,
-        props,
-        title: 'Verify your email address'
+        pane: VerifyEmailPane
       },
       // eslint-disable-next-line sort-keys
       terms: {
         disableNext: !hasConsentedToTerms,
         nextId: 'notifications',
         onNext: this._handleCreateNewUser,
-        pane: TermsOfUsePane,
-        props,
-        title: 'Create a new account'
+        pane: TermsOfUsePane
       },
       // eslint-disable-next-line sort-keys
       notifications: {
         disableNext: notificationChannel === 'sms' && !userData.phoneNumber,
         nextId: 'places',
         pane: NotificationPrefsPane,
-        prevId: 'terms',
-        props,
-        title: 'Notification preferences'
+        prevId: 'terms'
       },
       places: {
         nextId: 'finish',
         pane: FavoritePlacesList,
-        prevId: 'notifications',
-        props,
-        title: 'Add your locations'
+        prevId: 'notifications'
       },
       // eslint-disable-next-line sort-keys
       finish: {
         pane: AccountSetupFinishPane,
-        prevId: 'places',
-        props,
-        title: 'Account setup complete!'
+        prevId: 'places'
       }
     }
 
+    // Pass props and title to each entry above
+    Object.keys(paneSequence).forEach(k => {
+      const pane = paneSequence[k]
+      pane.props = this.props
+      pane.title = <FormattedMessage id={`components.NewAccountWizard.${k}`} />
```

Comment:
Please do not use variables as message IDs as it is hard to automatically keep track of which IDs are in the code. Instead, please create a hardcoded message ID string for each possible message.